### DISCUSSION TITLE
Audio sounds one-pitch down in streaming.

### DISCUSSION THREAD

**Discussion by JamesJunsungKim** [HaishinKit/HaishinKit.swift#1156] [Q&A]
Hi, thank you for your response to my last two questions. Big appreciation.

But I have a one more situation where when streaming a video, it sounds one-pitch down. 

Is there a possible reason for this issue?

I set h_264_high_auto for a profile. 

I really appreciate any help you can provide.

**Comment by shogo4405** [comment]
I've never had an experience like that, so I'm not sure.

If you resolve this issue, you have to provide more informations like this template. https://github.com/shogo4405/HaishinKit.swift/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml

Thank you.
